cmd/rekor-server/app/root.go

```diff
@@ -61,6 +61,8 @@ func init() {
 	rootCmd.PersistentFlags().Uint16("trillian_log_server.port", 8091, "Trillian log server port")
 	rootCmd.PersistentFlags().Uint("trillian_log_server.tlog_id", 0, "Trillian tree id")
 	rootCmd.PersistentFlags().String("rekor_server.address", "127.0.0.1", "Address to bind to")
+	rootCmd.PersistentFlags().String("rekor_server.signer", "", "Rekor signer to use. Current valid options include: [gcpkms]")
```

Comment:
do we need to mention the memory signing option here (or make it the explicit default)?

Reply:
good idea, updated!